integrationtest/src/androidTest/java/com/swiftkey/cornedbeef/CoachMarkTestCase.java

```diff
@@ -121,7 +122,28 @@ public void testOnDismissCalled() throws InterruptedException {
         
         verify(mockListener, times(1)).onDismiss();
     }
-    
+
+    /**
+     * Verify that onTimeout is called when coach mark's timeout expired
+     * @throws InterruptedException
+     */
+    public void testOnTimeoutCalled() throws InterruptedException {
+        final long timeout = 1000;
```

Comment:
You will be holding up the tests for a second when you can control how long you want. Can we make this as small as possible whilst still allowing the UI to react etc.